lib/manager.js

```diff
@@ -82,14 +96,35 @@ Manager.prototype.load = function () {
       console.error('Manager load error: ' + e)
     }
 
-    self.getServers().map(function (server) {
-      if (server.auto_start) {
-        server.start()
-      }
-    })
+    self.getServers()
+      .filter(server => server.auto_start)
```

Comment:
This will make old Node.js versions go 💥

Reply:
do we really need to be compatible with NodeJS<4 :sob: